.github/workflows/unit_tests.yaml

```diff
@@ -0,0 +1,35 @@
+---
+name: Unit tests
+"on":
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
+jobs:
+  go-test:
+    runs-on: ubuntu-latest
+    container:
+      image: registry.access.redhat.com/ubi8/go-toolset:1.19.13-2.1698062273
```

Comment:
Shouldn't we be using go 1.20.10+ at this point to mitigate CVE-2023-44487? See [RHTAPWATCH-546](https://issues.redhat.com/browse/RHTAPWATCH-546) which predated this new exporter. cc @hmariset

Reply:
yes, `registry.access.redhat.com/ubi9/go-toolset:1.20.10-2` is the image we are using now in segment-bridge

Reply:
Updated